client/src/components/TrendingHashtagCard/TrendingHashtagCard.css

```diff
@@ -0,0 +1,40 @@
+.TrendingHashtagCard__container {
+  width: 100%;
+  height: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  font-family: var(--default-font);
+}
+.TrendingHashtagCard__container h1 {
+  position: absolute;
+  left: 5%;
+  color: var(--white);
+}
+.TrendingHashtagCard__container h2 {
+  position: absolute;
+  left: 34%;
+  top: 40%;
+  color: var(--white);
+}
+.TrendingHashtagCard__hashtags-container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  position: absolute;
+  left: 50%;
+  top: 65%;
+  width: 350px;
+  height: 200px;
+  transform: translate(-50%, -50%);
+}
```

Comment:
Just noticed the absolute positioning here - can you find a way to do this with flexbox? We typically want to avoid absolute positioning, except for special cases

Reply:
@ZakkMan removed all the absoulte positioning, let me know what you think!